src/nile/cli.py

```diff
@@ -117,18 +117,26 @@ def setup(signer, network):
 @click.argument("method", nargs=1)
```

Comment:
why the change in the permissions? (`100755` → `100644`)

Reply:
Not sure at what moment the permission was updated, to be honest. Fixed again to 755.